pkg/dtos/dto.go

```diff
@@ -0,0 +1,63 @@
+//
+// Copyright (C) 2022 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package dtos
```

Comment:
move this file to internal/pkg/keeper/dto , because it is for keeper internal usage only